### DISCUSSION TITLE
专业版能打开连上服务器 但是打不开被屏蔽的网站

### DISCUSSION THREAD

**Discussion by cyrilpetrilli** [getlantern/lantern-forum-cn#431] [General]
如题。我在应用内反馈了两次付上了诊断信息。技术支持给我自动回复了 两个反馈 编号分别是71313509和71272787。 请问一般几个工作日有恢复？谢谢🙏。加油破墙的斗士们，愿荣光照耀你们。

**Comment by MaxMKMax** [comment]
@cyrilpetrilli 您好. 有的, 我帮您反馈上去了. 前天开会了. 您放心, 我会再问问看团队. 希望他们能够突破. 感谢您的祝福! 我们不只要破墙. 还要把墙推倒!!!
